com.stansassets.scene-management/Editor/Window/Tabs/SettingsTab.cs

```diff
@@ -2,13 +2,14 @@
 using StansAssets.Plugins.Editor;
 using UnityEditor;
 using UnityEditor.UIElements;
-using UnityEngine;
 using UnityEngine.UIElements;
 
 namespace StansAssets.SceneManagement
 {
     public class SettingsTab : BaseTab
     {
+        readonly VisualElement m_Enums;
```

Comment:
Why m_Enums? Please fix me if a missunderstood.
```suggestion
        readonly VisualElement m_StackVisualizersRoot;
```